tinkerbell/stack/values.yaml

```diff
@@ -18,6 +18,8 @@ stack:
     port: 8080
     image: alpine
     downloadsDest: /opt/hook
+    persistentStorage: true
+    storageClass: ceph-filesystem
```

Comment:
I think we want this to be specified by the user and not default. We can add a check to one of the templates that verifies if `persistentStorage == true` then require `storageClass`.